flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FCU/A320_Neo_FCU.js

```diff
@@ -432,6 +432,24 @@ class A320_Neo_FCU_Speed extends A320_Neo_FCU_Component {
             this.onPull();
         } else if (_event === "SPEED_TOGGLE_SPEED_MACH") {
             this.onSwitchSpeedMach();
+        } else if (_event === "USE_PRE_SEL_SPEED") {
+            clearTimeout(this._resetSelectionTimeout);
+            SimVar.SetSimVarValue("K:SPEED_SLOT_INDEX_SET", "number", 1);
```

Comment:
Maybe this should be done after the `selectedValue` is set. In practice I don't think it will matter.

Reply:
I literally copy and pasted this code from the pull() event of the fcu. Sure the order variables are set can be changed, bit I honestly don't see why xD